src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -217,7 +217,8 @@ public static void RegisterMSBuildPath(string[] msbuildSearchPaths)
                 string msbuildExe = Path.Combine(path, "MSBuild.exe");
                 if (File.Exists(msbuildExe))
                 {
-                    if (FileVersionInfo.GetVersionInfo(msbuildExe).FileMajorPart < 17)
+                    FileVersionInfo ver = FileVersionInfo.GetVersionInfo(msbuildExe);
+                    if (ver.FileMajorPart < 17 || ver.FileMinorPart < 1)
```

Comment:
Is that OK? Won't this fail for future 18.0?

Reply:
In that case, shouldn't the check be `if (ver.FileMajorPart < 17 || (ver.FileMajorPart == 17 && ver.FileMinorPart < 1))`?
Otherwise it will still set the variables for 18.0 since `(ver.FileMinorPart < 1)` is true.

Reply:
🤦‍♂️

Yes. Thank you; you've been a big help with this.

Reply:
```suggestion
                    if (ver.FileMajorPart < 17 || (ver.FileMajorPart == 17 && ver.FileMinorPart < 1))
```